restrict.dat.inv

```diff
@@ -1,3 +1,17 @@
-# Restricted frequency table for Rankin Inuvik.
-# Insofar no restrictions
-default=12200
+# Restricted frequency table for Inuvik.
+# License no. 010631959-003 effective March 15 2021
+# The use of frequencies in the following bands is not authorized: 9.995 - 10.005 MHz, 13.360 - 13.410 MHZ,
+# 14.990 - 15.010 MHZ, 19.990 - 20.010 MHZ
+# The use of the following frequencies is prohibited: 8.0115 MHz, 8.0765 MHz, 8.117 MHz, 8.1674 MHz, 9.2124 MHz,
+# 9.4815 MHz, 10.1965 MHz, 13.4115 MHz, 15.7345 MHz, 15.7995 MHz
+# 0.205 MHz buffer added to account for emission bandwidth (200 kHz in license)
+# ** NOTE ** Although these restricted frequencies are the same as Saskatoon's, they are indeed
+# from the Inuvik radio license. **
+default=13700
+7806 8373
+9007 9687
+9790 10210
```

Comment:
I think that this line needs a higher buffer to account for 10.1965

Reply:
I've updated it as well as Saskatoon's